app/src/main/java/com/my/playlistmaker/domain/db/impl/PlaylistInteractorImpl.kt

```diff
@@ -0,0 +1,34 @@
+package com.my.playlistmaker.domain.db.impl
+
+import android.net.Uri
+import com.my.playlistmaker.data.db.PlaylistRepository
+import com.my.playlistmaker.domain.db.PlaylistInteractor
+import com.my.playlistmaker.domain.models.Playlist
+import com.my.playlistmaker.domain.models.Track
+import kotlinx.coroutines.flow.Flow
+
+class PlaylistInteractorImpl(private val playlistRepository: PlaylistRepository):PlaylistInteractor {
+    override suspend fun createPlaylist(cover: Uri?, name: String, description: String) {
+        playlistRepository.createPlaylist(cover, name, description)
+    }
```

Comment:
Подобные функции можно писать в 1 строку
```suggestion
    override suspend fun createPlaylist(cover: Uri?, name: String, description: String) = playlistRepository.createPlaylist(cover, name, description)  
```